cli/parse.go

```diff
@@ -67,6 +67,19 @@ func isRecursive(req *cmds.Request) bool {
 	return rec && ok
 }
 
+func useGitIgnoreFile(req *cmds.Request) bool {
```

Comment:
Rather than a flag that signals we should look for a ".gitignore" file, would it make sense to have a parameter with the path to the ignore file? eg
`--ignore-file FILENAME`

Reply:
Yes, that does make sense. I named the option `--ignore-rules`. I was concerned that `ignore-file` could be interpreted as a phrase. I'm open to alternative naming.